### PR TITLE
Changes to OtherCourseSettings model and adding an entrypoint from cms.

### PR DESCRIPTION
**Pull request by ivanvgh**
### Description:
This PR adds a migration file that changes CourseOverview field with CourseId in OtherCourseSettings and adds other_course_settings json blob in the model. 
Also is adding entrypoint from cms.

More context about this change can be found in [this ticket](https://pearsonadvance.atlassian.net/browse/PAE-604?atlOrigin=eyJpIjoiOGNkOTA0YTUzOTRkNGJlZjg4YmExNTY0Nzc3OTU0MTIiLCJwIjoiaiJ9).

### Reviewers:

- [ ] @diegomillan

### CODE REVIEW

File: openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py [hunk 1]
```diff
@@ -19,10 +18,11 @@ def update_course_settings(*args, **kwargs):  # pylint: disable=unused-argument
         other_course_settings = kwargs.get('other_course_settings', {})
 
         OtherCourseSettings.objects.update_or_create(
-            course_overview=CourseOverview.get_from_id(kwargs.get('course_key')),
+            course_id=kwargs.get('course_key'),
             defaults={
                 'external_course_id': other_course_settings.get('external_course_run_id', ''),
```
Comment (diegomillan) [inline openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py:23]: I rather to use 
```suggestion
                'external_course_id': other_course_settings.get('external_course_run_id'),
```
Leaving those fields as null values on DB if not present.

File: openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py [hunk 2]
```diff
@@ -19,10 +18,11 @@ def update_course_settings(*args, **kwargs):  # pylint: disable=unused-argument
...
                 'external_platform': other_course_settings.get('external_platform_target', ''),
```
Comment (diegomillan) [inline openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py:24]: Same as the line above: 
```suggestion
                'external_platform': other_course_settings.get('external_platform_target'),
```

**Review by diegomillan — CHANGES_REQUESTED** [review]

File: openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py (hunk 2 above)
Comment (ivanvgh) [reply openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py:24]: Applied.

File: openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py (hunk 1 above)
Comment (ivanvgh) [reply openedx_external_enrollments/edxapp_wrapper/backends/course_settings_j_v1.py:23]: Applied, since this field is nullable.

**Review by diegomillan — APPROVED** [review]

**Pull request merged**